docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: docs
+title: Importing projects
```

Comment:
This docs talks about _Importing_ projects, but in the IDEs, the action is called to as _Add Existing Projects_

Reply:
Changed to `Add Existing Projects` throughout.